src/config.h.in

```diff
@@ -21,7 +21,7 @@ constexpr auto PROJECT_NAME = "@PROJECT_NAME@";
 /**
  * Name of package
  */
-constexpr auto PROJECT_PACKAGE = "@PROJECT_PACKAGE@";
+constexpr auto PROJECT_PACKAGE = "@PROJECT_NAME@";
```

Comment:
This is not really great. Rename the target variable as well.

Reply:
The target variable already exists...

Reply:
I just removed it